grids/grids.css

```diff
@@ -0,0 +1,33 @@
+.container-grid {
+    width: 50%;
+}
+
+/* for multiple items in grid, uncomment the two lines below */
+.two-item-grid {
+    display: grid;
+    grid-gap: 10px;
+    /* grid-template-columns: repeat(2, 1fr); */
```

Comment:
Hey, you can remove these comments.

Reply:
Hey Namrata, I need those comments for future. Right now there are two items only in the grid, when we have more items, users can uncomment this and choose the number of columns. I want to leave the comment there. 
Example, if user wants to arrange 15 items in 3 columns, this comment will be useful.

Reply:
Removed anyways!